subsys/bluetooth/controller/ll_sw/ll_adv.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2017 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+struct ll_adv_set {
+	u8_t  chl_map:3;
+	u8_t  filter_policy:2;
+#if defined(CONFIG_BLUETOOTH_CONTROLLER_PRIVACY)
+	u8_t      rl_idx : 5;
+	u8_t      own_addr_type : 2;
+	u8_t      id_addr_type : 1;
+	u8_t      id_addr[BDADDR_SIZE];
```

Comment:
@Vudentz I'm not sure I follow. Bitfields themselves are endianness-neutral aren't they? Those are not packets coming over the air, those are for local storage only. I do agree that packets over the air would require to be careful about the host CPU's endianness, but not storage?